jax/lib/__init__.py

```diff
@@ -23,7 +23,7 @@
 import jaxlib
 
 # Must be kept in sync with the jaxlib version in build/test-requirements.txt
-_minimum_jaxlib_version = (0, 1, 55)
+_minimum_jaxlib_version = (0, 1, 58)
```

Comment:
Do we really need to change this, or is it enough to use a new version of tf-nightly? In other words, does the jnp.bfloat16 from jaxlib 0.1.55 match the tf.bfloat16 from tf-nightly?

Reply:
No, you're right. I thought I recalled we mentioned this was needed before, but it seems it is not. Thanks!

Reply:
There have been some problems with 0.1.58 (#5374). I am reluctant to force people to upgrade at the moment.

Reply:
I am going to be merging now, since minimum jaxlib was bumped to 1.59